website/docs/cloud-docs/api-docs/changelog.mdx

```diff
@@ -9,6 +9,11 @@ description: >-
 
 Keep track of changes to the API for HCP Terraform and Terraform Enterprise.
 
+## 2024-06-20
+* Add documentation for new [team tokens](/terraform/cloud-docs/api-docs/organization-team-tokens) API that lists team tokens in an organization.
+* Add reference to organization team token api on teams, index and navigation page.
+* Add a `sort` parameter to the [Team Tokens list API](/terraform/cloud-docs/api-docs/organization-team-tokens#query-parameters) to allow sorting tokens by team name, created by, expired at and last used.
```

Comment:
I also don't think we need this bullet point because it's covered by the first bullet point. 
```suggestion
```